python/dllib/src/bigdl/dllib/nncontext.py

```diff
@@ -743,11 +742,12 @@ def init_env(conf):
     os.environ["KMP_BLOCKTIME"] = kmp_blocktime
 
 
-def init_spark_conf(conf=None):
+def init_spark_conf(conf=None, **kwargs):
     spark_conf = SparkConf()
     if conf:
         spark_conf.setAll(conf.items())
-    init_env(spark_conf)
+    if "is_orca" is not in kwargs or kwargs["is_orca"] is False:
+        init_env(spark_conf)
```

Comment:
We need to properly set the ENV variables for Orca too

Reply:
We already have corresponding settings in the runner for TensorFlow and PyTorch.
- TensorFlow 1.x: [link](https://github.com/intel-analytics/BigDL/blob/fa0741d53cf7cd4179e2f852d63416229a6928a2/python/orca/src/bigdl/orca/learn/tf/tf_runner.py#L86)
- TensorFlow 2.x: [link](https://github.com/intel-analytics/BigDL/blob/fa0741d53cf7cd4179e2f852d63416229a6928a2/python/orca/src/bigdl/orca/learn/tf2/tf_runner.py#L256)
- PyTorch: [link](https://github.com/intel-analytics/BigDL/blob/main/python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py#L166)

Reply:
> We already have corresponding settings in the runner for TensorFlow and PyTorch.
> 
> * TensorFlow 1.x: [link](https://github.com/intel-analytics/BigDL/blob/fa0741d53cf7cd4179e2f852d63416229a6928a2/python/orca/src/bigdl/orca/learn/tf/tf_runner.py#L86)
> * TensorFlow 2.x: [link](https://github.com/intel-analytics/BigDL/blob/fa0741d53cf7cd4179e2f852d63416229a6928a2/python/orca/src/bigdl/orca/learn/tf2/tf_runner.py#L256)
> * PyTorch: [link](https://github.com/intel-analytics/BigDL/blob/main/python/orca/src/bigdl/orca/learn/pytorch/torch_runner.py#L166)
> 
> Do we need to explore the possibility of unifying these settings？

1) These only set the config inside respective library, not the global environment variables
2) Environment variables should be set before starting the JVM or Python process, not inside hte process
3) And yes, we need to refactor all the settings of configs and environment variables to make it more robust

Reply:
Why do we want to set the global environment variables? Is it more reasonable to set in each library so that if we run two workloads in one context, they won't impact each other?